src/lib.rs

```diff
@@ -115,15 +120,64 @@ fn print_notice(name: &str, current_version: &str, latest_version: &str) {
     print!("{}", generate_notice(name, current_version, latest_version));
 }
 
-/// Validates current version of crate
-/// Takes the current name and version
-/// Prints directly to stdout (Will probably change to be more asynchrounos)
-pub fn check_version(name: &str, current_version: &str) -> Result<(), Box<dyn std::error::Error>> {
+#[derive(Deserialize, Serialize)]
+struct Config {
+    last_checked: DateTime<Utc>,
+}
+
+fn get_app_name(name: &str) -> String {
+    let mut app_name: String = String::from(name);
+    app_name.push_str("-update-notifier");
+    #[cfg(test)]
+    app_name.push_str("-test");
+    app_name
+}
+
+fn update_time(date_time: DateTime<Utc>, name: &str) -> Result<(), Box<dyn std::error::Error>> {
+    let config = Config {
+        last_checked: date_time,
+    };
+    let config_store = Configstore::new(&get_app_name(name), AppUI::CommandLine)?;
+    config_store.set("config", config)?;
+    Ok(())
+}
+
+fn compare_with_latest(
+    name: &str,
+    current_version: &str,
+) -> Result<(), Box<dyn std::error::Error>> {
     let latest_version = get_latest_version(name)?;
     if latest_version != current_version {
         print_notice(name, current_version, &latest_version);
     }
-    Ok(())
+    let date_time = Utc::now();
+    update_time(date_time, name)
+}
+
+/// Validates current version of the crate
```

Comment:
I filed one of the bugs for this, we should add an example into this doc comment.